apps/web/app/routes/examples/forms/multiple-forms.tsx

```diff
@@ -0,0 +1,146 @@
+import hljs from 'highlight.js/lib/common'
+import type {
+  ActionFunction,
+  LoaderFunction,
+  MetaFunction,
+} from '@remix-run/node'
+import { formAction } from '~/formAction'
+import { z } from 'zod'
+import Form from '~/ui/form'
+import { metaTags } from '~/helpers'
+import { makeDomainFunction } from 'domain-functions'
+import Example from '~/ui/example'
+
+const title = 'Multiple Forms'
+const description =
+  'In this example, we show how you can handle multiple forms in the same route.'
+
+export const meta: MetaFunction = () => metaTags({ title, description })
+
+const code = `const loginSchema = z.object({
+  _action: z.enum(['/login', '/contact']),
```

Comment:
Couldn't this be a `z.literal`?

Reply:
Maybe I'm misunderstanding, but if I want the `_action` to allow both of these values, how can `z.literal` do that?

Reply:
On the route component, you're already passing a literal value to each form. Each form only accepts only one value. There's no form that can deal with both.

Then, on the action you're only calling `formAction` after you already know the value of the `_action` field. The only moment in time where it could be both is before parsing the zod schema, so it won't be an issue to use a literal.

Reply:
OH, I see now. Great point! I'll update them.